packages/api/src/hooks/useAllAvailableAccounts.ts

```diff
@@ -0,0 +1,26 @@
+import { useMemo } from 'react';
+import useGetAccountTypes from './useGetAccountTypes';
+import useLandingCompany from './useLandingCompany';
+
+/** A custom hook to get all available accounts that user can have. */
+const useAllAvailableAccounts = () => {
+    const { data: landing_company_data } = useLandingCompany();
+    const { data: account_types_data, ...rest } = useGetAccountTypes(
+        landing_company_data?.financial_company?.shortcode
```

Comment:
This means that we are removing the gaming_company from every API?

Reply:
Looks like it as per https://app.clickup.com/t/20696747/PLA-582